cordova-lib/spec-cordova/save.spec.js

```diff
@@ -244,19 +307,26 @@ describe('(save flag)', function () {
                 console.log(err);
                 expect(false).toBe(true);
                 done();
+            }).finally(function () {
+                revertDownloadPlatform();
             });
-        }, timeout);
+        }, TIMEOUT);
 
         it('spec.11 should update spec with git url when updating using git url', function (done) {
             helpers.setEngineSpec(appPath, platformName, platformVersionNew);
-            platform('add', platformName + '@' + platformVersionNew)
+            mockDownloadPlatform(platformLocalPathOld, platformVersionOld);
+
+            platform('add', platformName + '@' + platformVersionOld)
             .then(function () {
+                revertDownloadPlatform();
                 var fsExistsSync = fs.existsSync.bind(fs);
                 spyOn(fs, 'existsSync').andCallFake(function (somePath) {
                     return (somePath === path.join(appPath, 'platforms', platformName)) || fsExistsSync(somePath);
                 });
+                mockDownloadPlatform(platformLocalPathNew, platformVersionNew);
```

Comment:
Using edge for this test has caught 2 `cordova-android` issues in the last week or so. While ideally we don't want `cordova-lib` tests failing because of `cordova-android` issues, should we continue using edge in these tests until we've added `update` tests to `cordova-android`?